doc/source/programs/gdal_vector_geom_simplify_coverage.rst

```diff
@@ -0,0 +1,91 @@
+.. _gdal_vector_geom_simplify_coverage:
+
+================================================================================
+``gdal vector geom simplify-coverage``
+================================================================================
+
+.. versionadded:: 3.12
+
+.. only:: html
+
+    Simplify shared boundaries of a polygonal vector dataset.
+
+.. Index:: gdal vector geom simplify-coverage
+
+Synopsis
+--------
+
+.. program-output:: gdal vector geom simplify-coverage --help-doc
+
+Description
+-----------
+
+:program:`gdal vector geom simplify-coverage` simplifies shared boundaries of a
+polygonal dataset.
+
```

Comment:
Maybe the description could be improved. Actually also the outer boundary is simplified if not especially denied with `--preserve-external-boundary`. And what really is preserved is the topology of the coverage so that the shared boundaries remain shared, even they are not the same after simplification.

Reply:
... simplifies boundaries of a polygonal dataset, such that shared boundaries are preserved without introducing gaps or overlaps. Gaps or overlaps already present in the input dataset will not be corrected.

?

Reply:
Feels good.